packages/functional-tests/tests/misc/keyStretchingV2.spec.ts

```diff
@@ -14,6 +14,7 @@ import { RecoveryKeyPage } from '../../pages/settings/recoveryKey';
 import { SignupReactPage } from '../../pages/signupReact';
 import { TotpPage } from '../../pages/settings/totp';
 import { getCode } from 'fxa-settings/src/lib/totp';
+import { ConfigPage } from '../../pages/config';
```

Comment:
These changes are going to conflict with, https://github.com/mozilla/fxa/pull/16923. Might want to coordinate with Kat to see which changes make the most sense to land first.